packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -510,6 +507,21 @@ function makeAugmentedSchema(
         });
     });
 
+    function ensureNonEmptyInput(inputName: string | InputTypeComposer<any>) {
+        let input;
+        if (typeof inputName === 'string') {
+            input = composer.getITC(inputName);
+        } else {
+            input = inputName;
+        }
```

Comment:
Here could we consider using a ternary operator? :thinking: it'll be less verbose and input will be properly typed and const:

```suggestion
        const input = isString(inputName) ? composer.getITC(inputName) : inputName;
```

Also, in the suggestion I'm using the method `isString` from `import { isString } from "../utils/utils";`